nemo/utils/data_utils.py

```diff
@@ -46,18 +46,16 @@ def is_datastore_path(path) -> bool:
     """Check if a path is from a data object store.
     Currently, only AIStore is supported.
     """
-    return path.startswith('ais://')
+    return path.startswith('ais://') or path.startswith('s3://')
```

Comment:
can we change to `is_valid_url(path)` instead? You can use `from lhotse.utils import is_valid_url` or just paste

```python
def is_valid_url(value: str) -> bool:
    try:
        result = urlparse(value)
        return bool(result.scheme) and bool(result.netloc)
    except AttributeError:
        return False
```